examples/28_autocomplete_text_value_objects.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE HTML>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>JSONEditor | Auto Complete with Text/Value Objects</title>
+
+  <link href="../dist/jsoneditor.css" rel="stylesheet" type="text/css">
+  <script src="../dist/jsoneditor.js"></script>
+
+  <style type="text/css">
+    .editor-container {
+      width: 500px;
+      height: 400px;
+      margin-bottom: 30px;
+    }
+
+    p {
+      width: 800px;
+      font-family: "DejaVu Sans", sans-serif;
+      margin-bottom: 20px;
+    }
+
+    h3 {
+      font-family: "DejaVu Sans", sans-serif;
+      color: #333;
+      margin-top: 30px;
+      margin-bottom: 10px;
+    }
+
+    .demo-section {
+      margin-bottom: 40px;
+      padding: 20px;
+      border: 1px solid #ddd;
+      border-radius: 5px;
+      background-color: #f9f9f9;
+    }
+
+    .search-info {
+      background-color: #e8f4fd;
+      padding: 10px;
+      border-left: 4px solid #2196F3;
+      margin: 10px 0;
+      font-size: 14px;
+    }
+  </style>
+
+</head>
+<body>
+<h1>JSONEditor - Enhanced Autocomplete with Text/Value Objects</h1>
```

Comment:
I think examples work best when they demonstrate1 thing. Can you keep only example 2 from the three examples in this file?

I have the feeling that example 3 is supposed to demonstrate more than it does right now, it refers to entering a capital like "Washington", but that seems not to work. If this example really adds value over the 2nd example, can you create a separate example file demonstrating the feature you want to demonstrate with it?